packages/dd-trace/src/config.js

```diff
@@ -972,6 +968,25 @@ class Config {
     }
   }
 
+  _setSamplingRule (obj, name, value) {
+    if (value === null || value === undefined) {
+      return this._setValue(obj, name, null)
+    }
+
+    if (typeof value === 'string') {
+      value = value && value.split(',')
```

Comment:
we have modern javascript syntax now! enjoy it!
```suggestion
      value = value?.split(',')
```